src/fparser/Fortran2003.py

```diff
@@ -561,32 +561,51 @@ class BracketBase(Base):
 ::
     <bracket-base> = <left-bracket-base> <something> <right-bracket>
     """
+    @staticmethod
```

Comment:
Is there any reason for having this as a static method over a function? Presumably to minimise code changes?